doc/classes/Color.xml

```diff
@@ -101,7 +100,7 @@
 			<return type="Color" />
 			<param index="0" name="over" type="Color" />
 			<description>
-				Returns a new color resulting from blending this color over another. If the color is opaque, the result is also opaque. The second color may have a range of alpha values.
+				Returns a new color resulting from blending this color over the given color. If the color is opaque, the result is also opaque. [param over] may have any range of alpha values.
```

Comment:
> The second color may have a range of alpha values.

It really feels like this remark is entirely useless. I am not exactly sure what it even means.

Reply:
I think it means that alpha can be blended?

Reply:
I think it means overlaying [param over] on top of the color.

Reply:
Based on my testing, that's what it seems to be. Amusingly, this method may be a misnomer of sorts.